projects/i18n/interfaces/language.ts

```diff
@@ -172,10 +206,18 @@ export type LanguagePreview = {
     };
 };
 
+export interface TuiLanguagePreview extends LanguagePreview {}
+
+/**
+ * @deprecated use {@link TuiLanguage}
+ */
+// eslint-disable-next-line @typescript-eslint/naming-convention
 export interface Language
```

Comment:
add migration to
https://github.com/Tinkoff/taiga-ui/blob/d8a6bd53f197a0379f9016abc3d3253e0f54e02a/projects/cdk/schematics/ng-update/constants/types.ts#L1-L7

Reply:
fuxed